### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by Mikunj**
**Changes:**

- Updated all navigation headers so that they're consistent across both platforms.
   - This also fixed some nav layout issues on Android and iPhone X.
- Temporarily disabled code push since it was causing the app to crash on android.
   - If not needed we can get rid of it completely in the future.
- Added alternate text for onboarding non-aozora users.
- Fixed favourite category selection failing in onboarding.
- Onboarding shows the total time spent watching anime when user rates anime.
- Fixed parallel universe category typo.
- Disabled swipe right to change tabs.
- Fixed navigation of media in settings.
- Fixed notification badges getting clipped on android.
- Added global feed.
- Show loading indicator when creating post.
- Tapping comments on post takes user to the post view.

### CODE REVIEW

File: src/routes/tabs.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const Tabs = TabNavigator(
   {
     lazy: true,
     tabBarPosition: 'bottom',
-    swipeEnabled: Platform.OS === 'ios',
+    swipeEnabled: false,
```
Comment (NuckChorris) [inline src/routes/tabs.js]: OH GOD BLESS

File: src/App.js [hunk 2]
```diff
@@ -150,6 +150,7 @@ const ConnectedRoot = connect(({ feed }) => ({
 }))(RootContainer);
 
 // Check for Codepush only in production mode (Saves compile time & network calls in development).
-const wrapper = __DEV__ ? identity : codePush;
+// FIXME: Codepush is making android crash
+const wrapper = __DEV__ || Platform.OS === 'android' ? identity : codePush;
```
Comment (edencakir) [inline src/App.js]: Any ideas why android is crashing due to codepush?

File: src/screens/Search/SearchScreen/index.js [hunk 3]
```diff
@@ -127,14 +126,18 @@ class SearchScreen extends Component {
     }
   };
 
-  renderIndicator = () => <View />;
+  renderIndicator = () => null;
```
Comment (edencakir) [inline src/screens/Search/SearchScreen/index.js]: 🤔

File: src/store/onboarding/favoriteCategories.js [hunk 4]
```diff
@@ -65,7 +65,7 @@ export default [
     id: 52,
   },
   {
-    title: 'Paralel Universe',
+    title: 'Parallel Universe',
```
Comment (edencakir) [inline src/store/onboarding/favoriteCategories.js]: Oops!

**Pull request merged**